contracts/ecosystem/andromeda-message-bridge/src/contract.rs

```diff
@@ -0,0 +1,77 @@
+use andromeda_ibc::message_bridge::{ExecuteMsg, IbcExecuteMsg, InstantiateMsg, QueryMsg};
+
+use ado_base::ADOContract;
+use common::{ado_base::InstantiateMsg as BaseInstantiateMsg, error::ContractError};
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    to_binary, Binary, Deps, DepsMut, Env, IbcMsg, IbcTimeout, MessageInfo, Response, StdError,
+    WasmMsg,
+};
+use cw2::set_contract_version;
+
+const CONTRACT_NAME: &str = "crates.io:andromeda-message-bridge";
+const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn instantiate(
+    deps: DepsMut,
+    env: Env,
+    info: MessageInfo,
+    _msg: InstantiateMsg,
+) -> Result<Response, ContractError> {
+    set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
+
+    ADOContract::default().instantiate(
+        deps.storage,
+        env,
+        deps.api,
+        info,
+        BaseInstantiateMsg {
+            ado_type: "message-bridge".to_string(),
+            ado_version: CONTRACT_VERSION.to_string(),
+            operators: None,
+            modules: None,
+            primitive_contract: None,
+        },
+    )
+}
+
+#[cfg_attr(not(feature = "library"), entry_point)]
+pub fn execute(
+    _deps: DepsMut,
+    env: Env,
+    _info: MessageInfo,
+    msg: ExecuteMsg,
+) -> Result<Response, ContractError> {
+    match msg {
+        ExecuteMsg::SendMessage {
+            channel,
```

Comment:
Do you think we should store channel references in the contract? (i.e. this channel is for this chain etc) The message could then use a desired chain as opposed to a channel? This could be a separate message ofcourse.

Reply:
I think that's do explicit, it should probably just be a string so we have more flexibility